crates/net/eth-wire/src/errors/eth.rs

```diff
@@ -56,24 +56,37 @@ impl From<alloy_rlp::Error> for EthStreamError {
     }
 }
 
-/// Error variants that can occur during the `eth` sub-protocol handshake.
+/// Error  that can occur during the `eth` sub-protocol handshake.
 #[derive(thiserror::Error, Debug)]
-#[allow(missing_docs)]
+
 pub enum EthHandshakeError {
+    /// Error indicating a status message was received or sent outside of the handshake process.
     #[error("status message can only be recv/sent in handshake")]
     StatusNotInHandshake,
+    /// Error for receiving a non-status message during the handshake phase.
     #[error("received non-status message when trying to handshake")]
     NonStatusMessageInHandshake,
     #[error("no response received when sending out handshake")]
+    /// Error when no response is received during the handshake process.
     NoResponse,
     #[error(transparent)]
+    /// Error due to invalid fork data.
     InvalidFork(#[from] ValidationError),
     #[error("mismatched genesis in status message: {0}")]
+    /// Error indicating a mismatch in the genesis block during status exchange.
     MismatchedGenesis(GotExpectedBoxed<B256>),
     #[error("mismatched protocol version in status message: {0}")]
+    /// Error for mismatched protocol versions in status messages.
     MismatchedProtocolVersion(GotExpected<u8>),
     #[error("mismatched chain in status message: {0}")]
+    /// Error denoting a mismatch in chain details in status messages.
     MismatchedChain(GotExpected<Chain>),
     #[error("total difficulty bitlen is too large: got {got}, maximum {maximum}")]
-    TotalDifficultyBitLenTooLarge { got: usize, maximum: usize },
+    /// Error for excessively large total difficulty bit lengths.
```

Comment:
these are great,
let's make it more concise by rephrasing all of them slightly so we don't need the `Error for`

Reply:
> these are great, let's make it more concise by rephrasing all of them slightly so we don't need the `Error for`

ty lot, we are now avoiding totally Error